package SET_USER;

message CHAR_ID_X_Y
{
	optional int32 id = 1;
	optional int32 x = 2;
	optional int32 y = 3;
}

message CONTENTS
{
	repeated CHAR_ID_X_Y data = 1;
}